app/routes/api/github/get-getUserInfo.tsx

```diff
@@ -0,0 +1,7 @@
+import { Octokit } from "@octokit/core";
```

Comment:
why is the name get-getUserInfo?

Reply:
I followed the way of naming the other files in the api folder